.circleci/release_kmp.yml

```diff
@@ -281,6 +283,21 @@ jobs:
             TMP_WORKSPACE: *tmp-workspace
             ENV_VARS_FILE: *env-vars-file
 
+  generate-docs:
```

Comment:
The artifact generation is also running the dokka task to generate the docs https://github.com/twilio/twilio-verify-passkeys/blob/dev/shared/build.gradle.kts#L71. Is this job still necessary?

Reply:
seems to me dokka task is not invoked. [Log](https://app.circleci.com/pipelines/github/twilio/twilio-verify-passkeys/426/workflows/73323489-83fc-4282-aca1-0ca147c8bfc4/jobs/4612)